resources/translations/en_MEL.json

```diff
@@ -11,22 +11,30 @@
   "mel.datahub.multiselect.filter.placeholder": "",
   "mel.datahub.search.clear": "",
   "mel.datahub.search.filters.categoryKeyword": "",
+  "mel.datahub.search.filters.format": "",
```

Comment:
Is it normal that you don't even do the english translation.

Reply:
Yes, we do not do them at the moment as it's not part of the contract :slightly_smiling_face: